marklogic-data-hub-central/ui/src/components/entities/mapping/source-entity-map/source-to-entity-map.test.tsx

```diff
@@ -22,7 +22,7 @@ describe('RTL Source-to-entity map tests', () => {
         expect(getByText('Clear')).toBeDisabled;
         expect(getByTestId("entityContainer")).toBeInTheDocument();
         expect(getByTestId("srcContainer")).toBeInTheDocument();
-        expect(getByText("Unable to find source documents using the specified collection or query.")).toBeInTheDocument;
+        expect(getByText("Unable to find source records using the specified collection or query.")).toBeInTheDocument;
```

Comment:
This is good, I think "records" works well in this context. FWIW, that's the term I found in the field to be most comfortable to users, who often think of "documents" as being things they create with Word and Powerpoint.

Reply:
@rjrudin is the scope of this change marklogic-data-hub-central only?

Reply:
That's really a question for @yasht2010 , but I'm pretty sure the answer is "yes".